### ISSUE TITLE
Comex MPI-PT: nb_wait_for_handle Error

### ISSUE THREAD

**Issue by edoapra** [GlobalArrays/ga#349]
When using the MPI-PT Comex port, the following error might show up
```
{0} nb_wait_for_handle Error: all user-level nonblocking handles have been exhausted
{1} nb_wait_for_handle Error: all user-level nonblocking handles have been exhausted
```
This is due to the fact the value of `COMEX_MAX_NB_OUTSTANDING` is set to 8 in https://github.com/GlobalArrays/ga/blob/develop/comex/src-mpi-pt/comex_impl.h

Instead, by increasing `COMEX_MAX_NB_OUTSTANDING`, the error no longer occurs  
`export COMEX_MAX_NB_OUTSTANDING=12`

I will push a pull request for this soon

**Comment by bjpalmer** [comment]
Can we close this issue?

**Comment by ajaypanyala** [comment]
resolved via #350
